blockstream/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@chainlink/blockstream-adapter",
+  "version": "0.0.1",
+  "description": "Chainlink blockstream adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "ethgasstation"
```

Comment:
switch `ethgasstation` to `blockstream`?

Reply:
```suggestion
    "blockstream"
```